[PMK-5702] new-terms-of-service-eng-ptbr-for-http3 (#630)
aziontech/docs: src/content/docs/pt-br/pages/contratos/tos/index.mdx

**Edge Application:** projetado para que Clientes construam e executem suas próprias aplicações serverless no Edge da rede, mais próximo a usuários e dispositivos. É ideal para aplicações que exigem baixos tempos de resposta ou processamento de dados em tempo real. Cada Edge Application pode ser atrelado a múltiplos domínios, que por sua vez são acessíveis por usuários e dispositivos. Para garantir o bom funcionamento dos serviços, a Azion poderá habilitar, desabilitar ou modificar serviços, POPs, endereços IPs, servidores, qualquer equipamento e conteúdo do cliente, sem aviso prévio, sempre visando a manutenção ou melhora da performance e disponibilidade dos mesmos. Salvo disposição em contrário no Contrato de Cliente ou quaisquer documentos anexos a ele, a cobrança do Edge Application se dará pela quantidade de (a) Domains, (b) Edge Applications, (c) Total Data Transferred e (d) Requests, de acordo com os volumes consumidos, e pelo volume consumido por seus módulos.

> **Edge Caching:** módulo do Edge Application projetado para entregar conteúdos web de qualquer formato e tipo, inclusive HTTP Live Streaming (HLS), por meio dos protocolos HTTP/1.0, HTTP/1.1, HTTP/2 e HTTP/3 e com suporte aos métodos HTTP GET, HEAD e OPTIONS. Inclui acesso a (1) rede da Azion para entrega de conteúdo; (2) funcionalidade de cache de conteúdos com TTL mínimo de 60 segundos; (3) manipulação de cabeçalhos HTTP; (4) criação de regras para redirecionamento ou entrega do conteúdo; (5) ferramenta de Real-Time Purge; (6) entrega de conteúdo em cache enquanto a origem estiver inacessível; (7) entrega de conteúdo HTTPS (TLS) com suporte a OCSP Stapling; (8) upload de certificado digital e suporte ao formato TLS x509 SNI, caso suportado pelo browser do usuário, ou uso do certificado compartilhado no domínio da Azion; (9) integração com os Produtos e Serviços da Azion. Para utilizar este serviço, é necessário que o Cliente faça o armazenamento da versão original de Seu Conteúdo em um servidor web de origem com acesso por meio do protocolo HTTP. NÃO EXISTE UMA GARANTIA DE QUE O CONTEÚDO (OBJETO) ESTARÁ DISPONÍVEL EM CACHE DURANTE TODA DURAÇÃO DO TIME-TO-LIVE (TTL) CONFIGURADO PELO CLIENTE. A AZION FAZ OS INVESTIMENTOS NECESSÁRIOS PARA MANTER OS DADOS EM CACHE PELO TEMPO MAIS PRÓXIMO DO QUE FOR CONFIGURADO PELOS CLIENTES, PORÉM OS OBJETOS MENOS ACESSADOS PODEM SER AUTOMATICAMENTE REMOVIDOS DO CACHE PARA CEDER ESPAÇO PARA OBJETOS DE MAIOR VOLUME DE ACESSOS ATRAVÉS DE UM ALGORITMO DE LRU (LEAST RECENTLY USED). O TAMANHO MÁXIMO DE OBJETO É DE 10GB E, PARA UTILIZAR O SERVIÇO COM OBJETOS MAIORES É NECESSÁRIO AUTORIZAÇÃO DA AZION POR ESCRITO. Salvo disposição em contrário no Contrato de Cliente ou quaisquer documentos anexos a ele, a cobrança do Edge Caching se dará pelo (a) Total Data Transferred; (b) Requests; e (c) Real-Time Purge Operations, de acordo com os volumes consumidos.

Reviewer: ```suggestion
**Edge Caching:** módulo do Edge Application projetado para entregar conteúdos web de qualquer formato e tipo, inclusive HTTP Live Streaming (HLS), por meio dos protocolos HTTP/1.0, HTTP/1.1, HTTP/2 e HTTP/3 e com suporte aos métodos HTTP GET, HEAD e OPTIONS. Inclui acesso a (1) rede da Azion para entrega de conteúdo; (2) funcionalidade de cache de conteúdos com TTL mínimo de 60 segundos; (3) manipulação de cabeçalhos HTTP; (4) criação de regras para redirecionamento ou entrega do conteúdo; (5) ferramenta de Real-Time Purge; (6) entrega de conteúdo em cache enquanto a origem estiver inacessível; (7) entrega de conteúdo HTTPS (TLS) com suporte a OCSP Stapling; (8) upload de certificado digital e suporte ao formato TLS x509 SNI, caso suportado pelo browser do usuário, ou uso do certificado compartilhado no domínio da Azion; (9) integração com os Produtos e Serviços da Azion. Para utilizar este serviço, é necessário que o Cliente faça o armazenamento da versão original de Seu Conteúdo em um servidor web de origem com acesso por meio do protocolo HTTP. NÃO EXISTE UMA GARANTIA DE QUE O CONTEÚDO (OBJETO) ESTARÁ DISPONÍVEL EM CACHE DURANTE TODA DURAÇÃO DO TIME-TO-LIVE (TTL) CONFIGURADO PELO CLIENTE. A AZION FAZ OS INVESTIMENTOS NECESSÁRIOS PARA MANTER OS DADOS EM CACHE PELO TEMPO MAIS PRÓXIMO DO QUE FOR CONFIGURADO PELOS CLIENTES, PORÉM OS OBJETOS MENOS ACESSADOS PODEM SER AUTOMATICAMENTE REMOVIDOS DO CACHE PARA CEDER ESPAÇO PARA OBJETOS DE MAIOR VOLUME DE ACESSOS ATRAVÉS DE UM ALGORITMO DE LRU (LEAST RECENTLY USED). O TAMANHO MÁXIMO DE OBJETO É DE 10 GB E, PARA UTILIZAR O SERVIÇO COM OBJETOS MAIORES É NECESSÁRIO AUTORIZAÇÃO DA AZION POR ESCRITO. Salvo disposição em contrário no Contrato de Cliente ou quaisquer documentos anexos a ele, a cobrança do Edge Caching se dará pelo (a) Total Data Transferred; (b) Requests; e (c) Real-Time Purge Operations, de acordo com os volumes consumidos.
```

#### Versões anteriores deste documento:

> [Termo de Serviço – 18 de setembro de 2023](/pt-br/documentacao/contratos/tds/18-september-2023/)

Reviewer: ```suggestion
[Termo de Serviço – 18 de setembro de 2023](/pt-br/documentacao/contratos/tds/18-setembro-2023/)
```

> [Termo de Serviço – 18 de setembro de 2023](/pt-br/documentacao/contratos/tds/18-september-2023/)

Reviewer 2: ```suggestion
[Termo de Serviço – 18 de setembro de 2023](/pt-br/documentacao/contratos/tds/18-setembro-2023/)
```
